### PR TITLE
rm future

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_db/test/test_util.py [hunk 1]
```diff
@@ -1172,13 +1172,7 @@ def test_BytesIO(self):
         with qdb.util.open_file(f) as fh:
             self.assertTrue(fh is f)
 
-    def test_hdf5IO(self):
```
Comment (antgonza) [inline qiita_db/test/test_util.py:1175]: Not sure why we need this test but it seems like a duplicate of the one below ...

Comment (ElDeveloper) [reply qiita_db/test/test_util.py:1175]: Looks like this test is different from 1175, since here the test is checking for the identity of the filehandle, whereas the other test checks that the returned object is an instance of `File`. Maybe worth adding the assertion in 1178 to the test below.

**Comment by ElDeveloper** [comment]
Ooops, looks like my local repo was very outdated. Future was already removed from the installation instructions and from the travis config.

**Comment by antgonza** [comment]
AFAIK we shouldn't have any issues with missing files/templates but hard to confirm a negative ... 

BTW I didn't use any of those tools; however, just tried to install drop2 and ran into some weird errors in the installation and wasn't sure how to run pyupgrade. 

I'll update the test.

**Comment by ElDeveloper** [comment]
Thanks @antgonza!
